[ADD] ecommerce: new payment providers page (#3346)
odoo/documentation: content/applications/websites/ecommerce/checkout_payment_shipping/payments.rst

=================
Payment providers
=================

Odoo supports a multitude of online

> :doc:`payment providers <../../../../applications/finance/payment_providers>` for your website,

Reviewer: My previous comment about unnecessary long refs applies to this `doc` role too.

> The default payment method for the eCommerce app is the **wire transfer**. Although not mandatory,
> it is strongly recommended to add additional payment providers for your customers. Odoo supports a
> multitude of online :doc:`payment providers <../../../../applications/finance/payment_providers>`
> for your website.

Reviewer 2: ```suggestion
The default payment method for the eCommerce app is **wire transfer**. Although not mandatory,
adding additional payment methods for your customers is strongly recommended. Odoo supports a
multitude of online :doc:`payment providers <../../../../applications/finance/payment_providers>`
for your website.
```
- wire transfer (I think no _the_ before)
- adding additional payment **methods**

.. seealso::
   :doc:`../../../../applications/finance/payment_providers`

Configuration
-------------

> To setup payment providers on the eCommerce app, go to :menuselection:`Website --> Configuration -->

Reviewer 2: ```suggestion
To set up payment providers on the eCommerce app, go to :menuselection:`Website --> Configuration -->
```
[Setup is one word when it is a noun or an adjective. It is two words when it functions as a verb.](https://grammarist.com/spelling/set-up-vs-setup/)

Payment Providers`. From here, :guilabel:`Activate` the payment providers you wish to have available
on your shop, and configure them according to your needs.

> Alternatively, you can also access **payment providers** via :menuselection:`Website -->

Reviewer 2: ```suggestion
Alternatively, you can access **payment providers** via :menuselection:`Website -->
```

Debit` if you wish to use it, as well as :guilabel:`View other providers`. If you use the
:guilabel:`Authorize.net` payment provider, the
:ref:`Payment Capture Method <payment_providers/features/manual_capture>` can be configured in that
same menu.

> If you are using :doc:`../../../finance/payment_providers/paypal`, you can also enable and configure

Reviewer: Here too, but in this case it's correct like this if you prefer to use relative refs over absolute refs.

the :guilabel:`Paypal - Express Checkout` feature in this menu.

Checkout payment options
------------------------

> Once activated, customers can choose the payment provider of their choice during the **checkout

Reviewer 2: payment method?

Author: After checking with @AntoineVDV, it is indeed a payment provider. Might change in the future :)

=================
Payment providers
=================

> The default payment method for the eCommerce app is **wire transfer**. Although not mandatory,

Reviewer: Wire transfer is no longer installed by default in Odoo 16.

adding additional payment providers for your customers is strongly recommended. Odoo supports a
multitude of online :doc:`payment providers <../../../../applications/finance/payment_providers>`
for your website.

.. seealso::

>    :doc:`../../../../applications/finance/payment_providers`

Reviewer: ```suggestion
   :doc:`../../../finance/payment_providers`
```
or
```suggestion
   :doc:`/applications/finance/payment_providers`
```

Configuration
-------------

To set up payment providers on the eCommerce app, go to :menuselection:`Website --> Configuration
--> Payment Providers`. From here, :guilabel:`Activate` the payment providers you wish to have
available on your shop, and configure them according to your needs.

Alternatively, you can access **payment providers** via :menuselection:`Website --> Configuration
--> Settings`. In the :guilabel:`Shop - Payment` section, you can :guilabel:`Configure SEPA Direct
Debit` if you wish to use it, as well as :guilabel:`View other providers`. If you use the
:guilabel:`Authorize.net` payment provider, the
:ref:`Payment Capture Method <payment_providers/features/manual_capture>` can be configured in that
same menu.

If you are using :doc:`../../../finance/payment_providers/paypal`, you can also enable and configure

> the :guilabel:`Paypal - Express Checkout` feature in this menu.

Reviewer: The "Paypal express checkout" feature doesn't exist (yet).

@pierre-pde You promised to remove it!
